src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -97,7 +115,9 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
 
         setupToolbarItems()
         if let touchHelperView = touchHelperView {
-            view.insertSubview(touchHelperView, aboveSubview: webView)
+            if webView != nil {
+                view.insertSubview(touchHelperView, aboveSubview: webView!)
```

Comment:
This will still produce a linter warnings because of force unwrapping. Use `if let` instead.